package.json

```diff
@@ -53,6 +53,7 @@
     "prepublish": "in-publish && yarn run build || :",
     "postpublish": "in-publish && npm-run-all new-version-pr-generator",
     "dev": "npm-run-all copy-polaris-tokens storybook",
+    "dev-no-splash": "DISABLE_SPLASH=1 yarn run dev",
```

Comment:
Not sure this warrants adding a new script – it might be okay to tell people to run `DISABLE_SPLASH=1 yarn run dev`?

Reply:
This is the same reason as above, I wanted to document it so that people can find the option easily.

If we don't add a command, how many people are going to dig into the code to figure out they can set an environment variable? Probably not a lot

Reply:
The console message and Splash's README do a great job telling people about the `DISABLE_SPLASH` trick.

I see your point about adding even more discoverability, however having 2 ways of achieving the same thing means we have to document and provide support/answer questions about both.

Reply:
What's the support we need to provide for a yarn command? Seems fairly straightforward to me

Reply:
>What's the support we need to provide for a yarn command?

When yarn splash fails or needs to be disabled for any reason, we'd be giving people 3 ways of disabling it:

1. `DISABLE_SPLASH=1 yarn dev`
1. `DISABLE_SPLASH=1` environment variable + `yarn dev`
1. `yarn dev-no-splash` (probably doesn't work in Windows, btw)

It's not a lot of overhead, but it adds to the overall cognitive load for contributors.

Also, if people start using `yarn dev-no-splash` and we want to remove it later, we'll be disrupting their developer experience (and I really don't want us to have to think about a deprecation path for custom yarn scripts).

Plus, that's less documentation for us to write and keep up to date 🎉 

PS: I agree with the intent behind having it more discoverable, but I'm not sure it's appropriate to do it that way, especially since the console message & README doc are great ways to advertise this feature.